android/.project

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Useless?

Reply:
I think it and other element marked as useless had been included with the build on android 5.0. Can it be? 

About the os, on android 5 there is a bug on the header. It has white background and white text:
<img width="352" alt="image" src="https://user-images.githubusercontent.com/38431762/41457670-a7b556cc-7084-11e8-944a-a3d0d4df8dbb.png">

Reply:
Are you using Eclipse to open the Android project?

Reply:
no, I use vs code. Last days I used Android studio to test on virtual device or my real smartphone. I will try manually removing these code lines